src/main/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -0,0 +1,106 @@
+import { DraftClaimantResponse } from 'features/claimant-response/draft/draftClaimantResponse.ts'
+import { ResponseRejection } from 'claims/models/response/core/responseRejection.ts'
+import { FormaliseRepaymentPlanOption } from 'claimant-response/form/models/formaliseRepaymentPlanOption'
+import { CourtDetermination } from 'claims/models/response/core/courtDetermination.ts'
+import { ClaimantResponse } from 'claims/models/response/core/claimantResponse'
+import { FreeMediationOption } from 'response/form/models/freeMediation'
+import { ResponseAcceptance } from 'claims/models/response/core/responseAcceptance'
+import { YesNoOption } from 'models/yesNoOption'
+import { FormaliseRepaymentPlan } from 'claimant-response/form/models/formaliseRepaymentPlan'
+import { PaymentIntention as PaymentIntentionDraft } from 'shared/components/payment-intention/model/paymentIntention'
+import { PaymentIntention } from 'claims/models/response/core/paymentIntention'
+import { PaymentOption } from 'claims/models/paymentOption'
+import { PaymentSchedule } from 'claims/models/response/core/paymentSchedule'
+import { PaymentOption as PaymentOptionDraft, PaymentType } from 'shared/components/payment-intention/model/paymentOption'
+import { PaymentDate } from 'shared/components/payment-intention/model/paymentDate'
+import { Moment } from 'moment'
+import { MomentFactory } from 'shared/momentFactory'
+import { RepaymentPlan } from 'claims/models/response/core/repaymentPlan'
+
+export class ClaimantResponseConverter {
+
+  public static covertToClaimantResponse (draftClaimantResponse: DraftClaimantResponse): ClaimantResponse {
+
+    if (draftClaimantResponse.settleAdmitted && draftClaimantResponse.settleAdmitted.admitted === YesNoOption.NO) {
+      let reject: ResponseRejection = new ResponseRejection()
+      if (draftClaimantResponse.paidAmount) {
+        reject.amountPaid = draftClaimantResponse.paidAmount.amount
+      }
+      if (draftClaimantResponse.freeMediation) {
+        reject.freeMediation = draftClaimantResponse.freeMediation.option === FreeMediationOption.YES
+      }
+      if (draftClaimantResponse.rejectionReason) {
+        reject.reason = draftClaimantResponse.rejectionReason.text
+      }
+      return reject
+    }
+    else if (draftClaimantResponse.formaliseRepaymentPlan && this.getFormaliseOption(draftClaimantResponse.formaliseRepaymentPlan)) {
+      return this.createResponseAcceptance(draftClaimantResponse)
+    } else throw new Error('Unknown state of draftClaimantResponse')
+  }
+
+  private static createResponseAcceptance (draftClaimantResponse: DraftClaimantResponse): ResponseAcceptance {
+    const respAcceptance: ResponseAcceptance = new ResponseAcceptance()
+    if (draftClaimantResponse.paidAmount) {
+      respAcceptance.amountPaid = draftClaimantResponse.paidAmount.amount
+    }
+    respAcceptance.formaliseOption = this.getFormaliseOption(draftClaimantResponse.formaliseRepaymentPlan)
```

Comment:
Don't see the implementation for FormaliseOption to be as `REFER_TO_JUDGE`?

Reply:
There was no REFER_TO_JUDGE in the existing FormaliseRepaymentPlanOption class. I have now added it.